src/java/com/rapleaf/jack/BaseDatabaseConnection.java

```diff
@@ -51,10 +51,11 @@ public Connection resetConnection() {
    * performed as an attempt to recover from an exception
    */
   public Connection resetConnection(Throwable cause) {
-    LOG.warn("Resetting database connection to attempt to recover from exception: {}", cause);
```

Comment:
That's unrelated but was confusing as the brackets will always be printed in the logs.